### ISSUE TITLE
Body Date Text

### ISSUE THREAD

**Issue by nonsponsored** [nonsponsored/BeBestNDS#4]
### Site

americabydesign.gov

### Page URL

https://americabydesign.gov

### Type of Issue

Accessibility

### Description of Issue

The text color of 'Aug 21, 2025' doesn't meet color contrast requirements. Has a 1.43 ratio and fails AA & AAA Normal & Large.

### Device Type (Optional)

None

### Operating System (Optional)

None

### Browser (Optional)

None

### Confirmation

- [x] I confirm this is a factual report of a technical, usability, or accessibility issue.
